src/Pos/Binary/Ssc/GodTossing/Toss.hs

```diff
@@ -0,0 +1,37 @@
+-- | Binary instances for Toss types.
+
+module Pos.Binary.Ssc.GodTossing.Toss
```

Comment:
I guess the purpose of `Binary.*` packages is to make things more obvious for party X who will be implementing this protocol, so we should avoid lots of modules if it's possible. Cyclic dependency?